### DISCUSSION TITLE
Feature Request: Feeding Volume Calculator

### DISCUSSION THREAD

**Discussion by TaddyJ** [babybuddy/babybuddy#371] [Ideas]
Something that I think that would be super easy to add and would be very handy since there is already weight being tracked is a Feeding Volume Calculator.

The paediatrician has recommended feeding between 160-180mls per KG per day.
Currently our boy is just under 4kgs so to make it easy lets say 4kgs.

So I imagine this is how it would look like.
Overall Volume for feeding calc in mls ( 'OV' Found in the settings maybe?)
Ideal number of feeds per day ( ' N' Found in settings maybe?)
Weight ( 'W' ) Taken from Updated/Logged weights already in the app?
Calculation for per feed volume ( 'FV' ); FV = OV x W / N

This is handy because in the first several months you need to continuously up their feeds.
Having it displayed based on information Baby Buddy already collects and tracks would make it a little easier.

**Comment by cdubz** [comment]
So far I have intentionally avoided dealing with units anywhere in the app so I think this would be a tall order.  This has come up a few times recently thought and has me thinking about how units could be introduced but that would be a blocker to this sort of feature, I think.

**Reply by TaddyJ** [reply]
That's fair.
You could still do it with generic numbers and not units. Mls/is doesn't matter. The math still checks out
